<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QMainWindow" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #2b2b2b;
	color: #e0e0e0;
}

QPushButton {
	background-color: #3c3c3c;
	color: #ffffff;
	border: 1px solid #555555;
}

QPushButton:hover {
	background-color: #4a4a4a;
}

QPushButton:pressed {
	background-color: #2a2a2a;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labDateTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala Text</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labTimerTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala Text</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Timer:</string>
    </property>
   </widget>
   <widget class="QFrame" name="line1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>145</y>
      <width>535</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 2px solid #ffffff;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labDate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>535</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala Text</family>
      <pointsize>36</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;dd/mm/yyyy hh : mm : ss&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="line2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>355</y>
      <width>270</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 2px solid #ffffff;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labTimer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>270</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala Text</family>
      <pointsize>36</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;00:00:00:000&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listDateFormats">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>145</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget { 
	background-color: #1e1e1e; 
	color: #ffffff;
	border: 1px solid #666666; 
	selection-background-color: #444477; 
	selection-color: #ffffff; 
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChange">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>60</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>60</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
